Sources/BranchSDK/BranchEvent.m

```diff
@@ -134,8 +134,11 @@ - (instancetype)initWithCoder:(NSCoder *)decoder {
     self = [super initWithCoder:decoder];
 	if (!self) return self;
 
-	self.serverURL = [decoder decodeObjectOfClass:NSString.class forKey:@"serverURL"];
-	self.eventDictionary = [decoder decodeObjectOfClass:NSDictionary.class forKey:@"eventDictionary"];
```

Comment:
What was the representation of this object without the data type enumeration?

Reply:
Nil